admission controller will crash on OpenShift 3.9

Env:

172.31.17.27

Version:
Federatorai-operator 0.2.0 + alameda 0.3.8
or
Federatorai-operator 0.2.1 + alameda 0.3.9

Log:
[root@node17127 ~]# oc logs -f admission-controller-68c48964db-c28fb
panic: initialization failed: get deployment or deploymentconfig owning admission-controller failed: get deployment or deploymentConfig owning pod failed: get owner references of resource extensions/v1beta1, Resource=namespaces/alameda/replicasets in namespace alameda failed: replicasets.extensions "admission-controller-68c48964db" is forbidden: User "system:serviceaccount:alameda:admission-controller" cannot get replicasets.extensions in the namespace "alameda": User "system:serviceaccount:alameda:admission-controller" cannot get replicasets.extensions in project "alameda"

goroutine 1 [running]:
github.com/containers-ai/alameda/admission-controller/cmd/app.glob..func2(0x2005380, 0x2031750, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/containers-ai/alameda/admission-controller/cmd/app/run.go:61 +0x2e7
github.com/containers-ai/alameda/vendor/github.com/spf13/cobra.(*Command).execute(0x2005380, 0xc00000a070, 0x0, 0x0, 0x2005380, 0xc00000a070)
	/go/src/github.com/containers-ai/alameda/vendor/github.com/spf13/cobra/command.go:762 +0x473
github.com/containers-ai/alameda/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2005120, 0xc000044118, 0x0, 0x136fc96)
	/go/src/github.com/containers-ai/alameda/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/containers-ai/alameda/vendor/github.com/spf13/cobra.(*Command).Execute(0x2005120, 0xc000141f88, 0xc000044118)
	/go/src/github.com/containers-ai/alameda/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/go/src/github.com/containers-ai/alameda/admission-controller/cmd/main.go:8 +0x2d

--- a/admission-controller/deployment.yaml
+++ b/admission-controller/deployment.yaml
@@ -40,6 +40,7 @@ rules:
       - list
       - patch
   - apiGroups:
+      - extensions
       - apps
     resources:
       - replicasets
@@ -48,6 +49,7 @@ rules:
       - get
       - list
   - apiGroups:
+      - extensions
       - ""
     resources:
       - replicationcontrollers

--- a/example/deployment/kubernetes/admission-controller/clusterrole.yaml
+++ b/example/deployment/kubernetes/admission-controller/clusterrole.yaml
@@ -27,6 +27,7 @@ rules:
     - list
     - patch
 - apiGroups:
+    - extensions
     - apps
   resources:
     - replicasets
@@ -35,6 +36,7 @@ rules:
     - get
     - list
 - apiGroups:
+    - extensions
     - ""
   resources:
     - replicationcontrollers

--- a/helm/alameda/charts/admission-controller/templates/clusterrole.yaml
+++ b/helm/alameda/charts/admission-controller/templates/clusterrole.yaml
@@ -31,6 +31,7 @@ rules:
     - list
     - patch
 - apiGroups:
+    - extensions
     - apps
   resources:
     - replicasets
@@ -39,6 +40,7 @@ rules:
     - get
     - list
 - apiGroups:
+    - extensions
     - ""
   resources:
     - replicationcontrollers

